Bottom: 76c676c34d47a7bf20c28ca59480fa9cb6e9800c
Top:    ddf7cda10bfcc7194399960132c23001ab48a76e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-17 11:00:11 -0500



---
diff --git a/roles/docker-mantisbt/defaults/main.yml b/roles/docker-mantisbt/defaults/main.yml
index 29727bd90..07eb1f502 100644
--- a/roles/docker-mantisbt/defaults/main.yml
+++ b/roles/docker-mantisbt/defaults/main.yml
@@ -18,8 +18,7 @@ mantisbt_db_user: novatech
 mantisbt_image_repo: https://github.com/novatechweb/docker-mantisbt.git
 mantisbt_image_name: '{{ docker_registry_username }}/mantisbt'
 
-mantisbt_db_image_name: 'mysql'
-mantisbt_db_image_tag: 5
+mantisbt_db_image_name: 'mysql:5'
 
 # the name of the container being started
 mantisbt_container_name: '{{ docker_name_prefix }}mantisbt'
